modules/p2/ding_entity_rating/ding_entity_rating.module

```diff
@@ -287,6 +304,9 @@ function template_preprocess_ding_entity_rating_display(&$variables) {
   $module_path = drupal_get_path('module', 'ding_entity_rating');
   drupal_add_css($module_path . '/css/ding_entity_rating.css');
   drupal_add_js($module_path . '/js/ding_entity_rating.js');
+  drupal_add_js(array(
+    'ding_entity_rating_get_limit' => variable_get('ding_entity_rating_get_limit', DING_ENTITY_RATING_DEFAULT_GET_LIMIT)
```

Comment:
Please nest settings under the module name so we do not pollute the global `Drupal.settings` object e.g. `Drupal.settings.ding_entity_rating.get_limit`.